packages/forms/src/model.ts

```diff
@@ -1869,7 +1869,9 @@ export class FormGroup extends AbstractControl {
   }
 
   /** @internal */
-  _reduceChildren(initValue: any, fn: Function) {
+  _reduceChildren<AccumulatorT>(
+      initValue: any,
+      fn: (acc: AccumulatorT, control: AbstractControl, name: string) => AccumulatorT) {
     let res = initValue;
```

Comment:
I'd propose trying tp mimic the `Array.reduce` typing for this call.
```suggestion
  _reduceChildren<T>(
      initValue: T,
      fn: (acc: T, control: AbstractControl, name: string) => T): T {
    let res = initValue;
```
The `initValue` and the return of the `fn` should be type-compatible.

Reply:
Good idea, fixed.